Common/Helpers/DocDbRestHelper.cs

```diff
@@ -344,15 +335,19 @@ public DocDbRestHelper(IConfigurationProvider configProvider)
         /// number of items per page</param>
         /// <param name="continuationToken">Optional: If the request will be made in pages, and you have a continuation token 
         /// from a previous page, this will ensure the next page begins at the right place</param>
-        private void BuildHeaders(WebClient webClient)
+        private WebClient BuildWebClient()
         {
+            var webClient = new WebClient();
+            webClient.Encoding = System.Text.Encoding.UTF8;
             webClient.Headers.Add("Content-Type", "application/json");
             webClient.Headers.Add("Accept", "application/json");
             webClient.Headers.Add("x-ms-version", "2015-04-08");
```

Comment:
x-ms-version string should be in web.config or constants so its easy to find/replace